### ISSUE TITLE
Create slides with this

### ISSUE THREAD

**Issue by yigakpoa** [clentafrica/education#7]
### We would like to explain Climate Change to African Children and use slides to do that. We intend for these slides to be accessible #A11Y.

1. **Weather Changes:**
   "Imagine how hot it feels when the sun shines all day. Climate change makes the Earth hotter all the time, not just in summer."

2. **Changing Seasons:**
   "You know how we expect rains during certain months? Climate change makes these rains come at different times or sometimes not at all."

3. **Animals in Danger:**
   "Think about how animals need water and food. Climate change makes it harder for them to find what they need, and some animals are disappearing."

4. **Less Food:**
   "Farmers grow food for us to eat, but climate change makes it harder for plants to grow, so there might be less food."

5. **More Storms:**
   "Climate change causes more storms and floods, which can destroy homes and crops."

6. **Melting Ice:**
   "Far away, there is ice that helps keep the Earth cool. Climate change is melting this ice, making the Earth warmer."

### Difference Between Clean and Polluted Air

**Clean Air Examples:**
1. Fresh mountain air with no factories nearby.
2. Air in a dense forest full of trees.
3. Seaside air with the scent of the ocean.
4. Rural area air with no cars or industries.
5. Countryside air in places with lots of grass and plants.
6. Air in a national park or nature reserve.
7. Air in small villages with little traffic.
8. Air after a fresh rain in a clean environment.
9. Air in a desert with no nearby cities.
10. Air in open savannas with minimal human activity.

**Polluted Air Examples:**
1. Air near busy roads with lots of cars.
2. Air in cities with many factories.
3. Air in industrial zones with chemical plants.
4. Air around construction sites with lots of dust.
5. Air in urban areas with heavy traffic congestion.
6. Air in areas burning trash or fields.
7. Air near mines or quarries.
8. Air in regions with forest fires.
9. Air near ports with many ships.
10. Air in crowded neighborhoods with lots of cooking smoke.

**Comment by yugnaik** [comment]
should i use ppt or canva to make this and create a folder and then pull request?

**Comment by yugnaik** [comment]
I am interested to contribute. can you tell me the exact procedure on how to submit the information

**Comment by yigakpoa** [comment]
Hi @yugnaik, Thank you for your interest. It’s preferable you use canva to make this, create a folder, & then a pull request. Please check our marketing style guidelines in this [assets.md](https://github.com/clentafrica/marketing-and-comms/blob/main/Assets.md) for the major fonts & color codes we use to ensure consistency. 

I will go ahead to assign this to you.

**Comment by yugnaik** [comment]
Ma'am,Are you able to see my pull request?
If no,I will try again as I am new to github

**Comment by yigakpoa** [comment]
Yes @yugnaik I have seen the pull request and merged it. Well done 💃

**Comment by yugnaik** [comment]
@yigakpoa Ma'am if any issue or anything to add please inform me
Thank you for giving me the opportunity

**Comment by yigakpoa** [comment]
Hi @yugnaik , quick follow up on this. Have you been able to create the slide for this? or should we assign to the @clentafrica/wg-design team?

**Comment by yugnaik** [comment]
Hello Ma'am 
Are you talking about create slides on explain Climate Change to African Children 
I had made a canva ppt on this and had shared the ppt also 
And u had merged it also I think
Are you talking abt this
Is there anything I was suppose to do
Sorry for the confusion

**Comment by yigakpoa** [comment]
@yugnaik 
Aha! I’m so sorry for the confusion. I have found the [pull request here:](https://github.com/clentafrica/education/pull/8 ). 

Please can you give me edit access to it to enable us change the colors and fonts to the suit CLENT Africa’s style guideline. 

@clentafrica/wg-design please can someone take a look at this too?

**Comment by yugnaik** [comment]
https://www.canva.com/design/DAGMPAzNSkc/VADbcbC2_EDQIMUfpmUb_w/edit?utm_content=DAGMPAzNSkc&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by breno02lucas** [comment]
Hello, everyone!

My name is Breno and I'm new here. @yigakpoa  sent an invitation to join the group. 
I'm a graphic designer's teacher and could help develop educational materials from issues I saw in the Clent/education area.

@yugnaik could I help u with this presentation?

**Comment by yugnaik** [comment]
@breno02lucas Sir sure
I have shared the link before
U can just take a look and modify as in where u feel it's needed
Thank you

**Comment by breno02lucas** [comment]
@yugnaik my pleasure!

**Comment by breno02lucas** [comment]
@yigakpoa could you tell me what the children's grade level?

**Comment by yigakpoa** [comment]
Thanks @yugnaik & @breno02lucas for the amazing work on this 👏. 

@breno02lucas I will loop in @clentafrica/wg-partnerships to tell us which grade they want this suited for.
